src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/commands.py

```diff
@@ -0,0 +1,55 @@
+cli_command(__name__, 'datalake store account firewall show', adls_format_path.format('firewall_rules_operations', 'FirewallRulesOperations', 'get'), cf_datalake_store_account_firewall)
+cli_command(__name__, 'datalake store account firewall delete', adls_format_path.format('firewall_rules_operations', 'FirewallRulesOperations', 'delete'), cf_datalake_store_account_firewall)
+
+# account trusted id provider operations
+cli_command(__name__, 'datalake store account trustedprovider create', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'create_or_update'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider update', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'update'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider list', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'list_by_account'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider show', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'get'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider delete', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'delete'), cf_datalake_store_account_trusted_provider)
+
+# filesystem operations
+cli_command(__name__, 'datalake store filesystem show', adls_custom_format_path.format('get_adls_item'))
+cli_command(__name__, 'datalake store filesystem list', adls_custom_format_path.format('list_adls_items'))
+cli_command(__name__, 'datalake store filesystem create', adls_custom_format_path.format('create_adls_item'))
+cli_command(__name__, 'datalake store filesystem append', adls_custom_format_path.format('append_adls_item'))
+cli_command(__name__, 'datalake store filesystem delete', adls_custom_format_path.format('remove_adls_item'))
+cli_command(__name__, 'datalake store filesystem upload', adls_custom_format_path.format('upload_to_adls'))
+cli_command(__name__, 'datalake store filesystem download', adls_custom_format_path.format('download_from_adls'))
+cli_command(__name__, 'datalake store filesystem download', adls_custom_format_path.format('download_from_adls'))
+cli_command(__name__, 'datalake store filesystem test', adls_custom_format_path.format('test_adls_item'))
+cli_command(__name__, 'datalake store filesystem preview', adls_custom_format_path.format('preview_adls_item'))
+cli_command(__name__, 'datalake store filesystem join', adls_custom_format_path.format('join_adls_items'))
+cli_command(__name__, 'datalake store filesystem move', adls_custom_format_path.format('move_adls_item'))
```

Comment:
Consider using "file" instead of "filesystem" since the operations you are performing are on files, not the entire filesystem itself. This also will be natural to users of the storage module.